lib/jose-jwe-webcryptographer.js

```diff
@@ -589,6 +590,7 @@ export class WebCryptographer {
       case "ES256":
       case "ES384":
       case "ES512":
+      case "ES256K":
```

Comment:
This is actually not needed for the fix to the unit tests, as it's foreshadowing @Mischala's next up in line to enable `secp256k1` signing. However, it won't hurt yet, either.